package.json

```
@@ -85,6 +85,6 @@
     "lint:js": "eslint --ignore-path .gitignore --ext .js,.jsx .",
     "lint:css": "stylelint --ignore-path .gitignore '**/*.css'",
     "lint": "run-p lint:*",
-    "postinstall": "cp -r \"$(npm explore @oacore/design -- pwd)/lib/assets\" ./public/design"
+    "postinstall": "rm -rf ./public/design/ && cp -r \"$(npm explore @oacore/design -- pwd)/lib/assets\" ./public/design"
```

I am not an expert here but I am not sure about `&&`. Chaining using `&&` fails if the first command fails (due to access error or any other one). If you still want the second one to make an attempt, prefer `;`.

I am not sure it would help because the only error I see is an access error which if happens, does it both commands.

```suggestion
    "postinstall": "rm -rf ./public/design/; cp -r \"$(npm explore @oacore/design -- pwd)/lib/assets\" ./public/design"
```

---

I think -f covers that case. If `rm` fails it doesn't make sense to run `cp` command because it's likely that it will fail too or it will copy it incorrectly.